### PR TITLE
Added a selectAndJumpToToday() function

### PR DESCRIPTION
**Pull request by jankcat**
Added a selectAndJumpToToday() function, that will select the current date, and then navigate to the current month

### CODE REVIEW

**Comment by jankcat** [comment]
Ref to issue https://github.com/christopherdro/react-native-calendar/issues/104

**Comment by jankcat** [comment]
@christopherdro can this go through?

File: components/Calendar.js [hunk 1]
```diff
@@ -137,6 +137,13 @@ export default class Calendar extends Component {
     }
     this.props.onDateSelect && this.props.onDateSelect(date ? date.format(): null );
   }
+  
+  selectAndJumpToToday() {
+    const today = new Date();
+    const newMoment = moment(this.state.currentMonthMoment).set('month', today.getMonth());
+    this.setState({ selectedMoment: today });
+    this.setState({ currentMonthMoment: newMoment });
```
Comment (christopherdro) [inline components/Calendar.js:145]: No need to call setState twice. Can set both with 1 call

**Comment by christopherdro** [comment]
@jankcat Sorry for the late response. Just left a comment for a small update.

**Pull request closed without merging**